docs/reference/module-options.md

```diff
@@ -0,0 +1,148 @@
+# Module configuration options
+
+Most Apostrophe modules, both core and project-level, can be configured with settings that influence functionality without having to write custom Javascript. The sections below describe the options available in all modules as well as those specific to certain module types.
```

Comment:
I would strike "both core and project-level". If the truth is "most", is it worthwhile to carve out the exceptions?